src/Parser/ParseResult.php

```diff
@@ -3,15 +3,16 @@
 namespace RecursiveTree\Seat\TreeLib\Parser;
 
 use Illuminate\Support\Collection;
+use JsonSerializable;
 use RecursiveTree\Seat\TreeLib\Helpers\DynamicProperties;
+use ReturnTypeWillChange;
```

Comment:
This does not seem to be used

Reply:
Sorry, it was an auto import from the IDE when I was adding the return types of functions. It proposed this but I removed it and forgot to remove the import. It is cleaned up